includes/class-newspack-newsletters.php

```diff
@@ -843,6 +843,11 @@ public static function sync_with_mailchimp( $post ) {
 				update_post_meta( $post->ID, 'mc_campaign_id', $mc_campaign_id );
 			}
 
+			// Prevent updating content of a sent campaign.
+			if ( ! in_array( $campaign_result['status'], [ 'sent', 'sending' ] ) ) {
```

Comment:
I think the logic is backwards?

```suggestion
			if ( in_array( $campaign_result['status'], [ 'sent', 'sending' ] ) ) {
```

Reply:
Um, yes, don't tell anyone! Fixed in c881d473bfbf9214dfa8a0b8260bd2ffe2a81293